packages/oscal-react-library/src/components/OSCALCatalogBaseline.tsx

```diff
@@ -1154,10 +1624,58 @@ export default function OSCALCatalogBaseline() {
                   },
                 }}
               >
-                <option value={"AL"}>AL</option>
-                <option value={"AR"}>AR</option>
-                <option value={"MD"}>MD</option>
-              </Select>
+                <option value="AL">Alabama</option>
+                <option value="AK">Alaska</option>
+                <option value="AZ">Arizona</option>
+                <option value="AR">Arkansas</option>
+                <option value="CA">California</option>
+                <option value="CO">Colorado</option>
+                <option value="CT">Connecticut</option>
+                <option value="DE">Delaware</option>
+                <option value="DC">District of Columbia</option>
```

Comment:
Is this the same `Select` field and options used in #927? I think we could refactor this code and reuse it, once one of these PRs are merged in.